.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
I think you should add .vscode in the gitignore

Reply:
Ohhh yes! you are right. I am fixing it ASAP

Reply:
coool but it didn't remove it, to remove it use `git rm -r .vscode --cached`